h2olog.cc

```diff
@@ -44,24 +45,44 @@ Other options:
     return;
 }
 
+static void make_timestamp(char *buf, size_t buf_len)
+{
+    time_t t = time(NULL);
+    struct tm tms;
+    localtime_r(&t, &tms);
+    const char *iso8601format = "%FT%TZ";
+    strftime(buf, buf_len, iso8601format, &tms);
+}
+
+static void infof(const char* fmt, ...)
+    __attribute__((format (printf, 1, 2)));
+
+static void infof(const char* fmt, ...)
+ {
```

Comment:
Indentation is strange. Do `clang-format -i h2olog.cc`

Reply:
Applied. Thanks 😭